website/pages/blog/type-safe-your-api.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: feTS - Enhancing Type Safety in REST API Development
+authors: tuvalSimha
+date: 2023-06-25
+tags: [typescript, openapi, rest, api, fets]
+description: Discover feTS - Empower your REST API development with type safety, efficiency, and seamless integration using TypeScript and OpenAPI. With Spotify API example.
+thumbnail: /blog-assets/type-safe-your-api/endpoint.png
+image: /blog-assets/type-safe-your-api/endpoint.png
+---
+
+## Introduction
+
+Developers who have experience working with REST API often encounter challenges such as lack of type safety, manual response parsing, and the absence of a unified interface for endpoints. These issues can hinder the efficiency and reliability of REST API development.
+
+In this blog post, we will explore a powerful tool called feTS that addresses these problems and provides solutions for more effective REST API development.
+
+## What Is feTS?
+feTS, which stands for "Framework-Embedded TypeScript," is a robust tool designed specifically for TypeScript projects. It simplifies the development and consumption of REST API while ensuring type safety. By leveraging OpenAPI specifications and TypeScript decorators, feTS allows developers to build and interact with REST API more efficiently, reducing the likelihood of runtime errors and improving code quality. Importantly, feTS is fully compatible with any existing REST API.
```

Comment:
isn’t feTS fetch + TypeScript?

> TypeScript decorators

it doesn't use decorators from what I know

You can also add that it's fully compatible as long as there's an OpenAPI spec